azbfs/zt_test.go

```diff
@@ -55,7 +55,7 @@ func getBfsServiceURL() azbfs.ServiceURL {
 func generateName(prefix string) string {
 	// These next lines up through the for loop are obtaining and walking up the stack
 	// trace to extract the test name, which is stored in name
-	pc := make([]uintptr, 10)
+	pc := make([]uintptr, 20)
```

Comment:
What is this for ?

Reply:
This is no longer necessary. I was having some issues with the pipeline passing and I thought it was cause this pc stack wasnt big enough. I resolved the issue in another way.  Will revert this